.travis.yml

```diff
@@ -6,26 +6,34 @@ before_install:
     - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1204-3.0.14.tgz
     - tar xvf mongodb-linux-x86_64-ubuntu1204-3.0.14.tgz
     - mv mongodb-linux-x86_64-ubuntu1204-3.0.14 3.0.14
-    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1204-3.2.11.tgz
-    - tar xvf mongodb-linux-x86_64-ubuntu1204-3.2.11.tgz
-    - mv mongodb-linux-x86_64-ubuntu1204-3.2.11 3.2.11
+    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1204-3.2.12.tgz
+    - tar xvf mongodb-linux-x86_64-ubuntu1204-3.2.12.tgz
+    - mv mongodb-linux-x86_64-ubuntu1204-3.2.12 3.2.12
+    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1204-3.4.2.tgz
+    - tar xvf mongodb-linux-x86_64-ubuntu1204-3.4.2.tgz
+    - mv mongodb-linux-x86_64-ubuntu1204-3.4.2 3.4.2
 
 script:
-    - mkdir -p ./data/db30 ./data/db30-ssl ./data/db32 ./data/db32-ssl
+    - mkdir -p ./data/db30 ./data/db30-ssl ./data/db32 ./data/db32-ssl ./data/db34 ./data/db34-ssl
```

Comment:
Added a tracking issue for Morc (this is a thing now. i'm making this name happen)